### DISCUSSION TITLE
[Feat] Ideas for productivity when working with raster space-time datasets and more

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [OSGeo/grass#2020] [Ideas]
Here https://gitlab.com/NikosAlexandris/tygrass/-/blob/6a0e13310c4a7a086dc74ee6c61c2fa467881a85/.grass.tygrass.strds are some functions I use in a day-to-day basis, always within the Terminology terminal, when working with time series.  Perhaps of interest or inspiration.

For example, `tyraster.strds.w.legend.title` will plot all raster maps registered in an STRDS with a horizonal legend and a title (if set).

**Comment by wenzeslaus** [comment]
I like the tygrass repo and browsed through it couple times in the past. For an enhancement issue, I think a more concrete proposal is needed to make it actionable. Hence, moving this to _Discussions > Ideas_.
